pegasus/sites.v3/code.org/public/teacher-dashboard/landing.haml

```diff
@@ -76,7 +76,10 @@ theme: none
       %h1{style: 'margin-bottom: 0px'}= I18n.t('dashboard_landing_available_title')
       %h2= I18n.t('dashboard_hoc_courses')
       = I18n.t('dashboard_hoc_courses_desc')
-    %br/  
+      %div{style: 'float:right; margin-right:30px; margin-top:-8px;'}
+        %a{:href=>'https://hourofcode.com'}
+          %button= I18n.t('learn_more')
```

Comment:
I think the text should be more descriptive like "Learn more about the Hour of Code" or "Join the Hour of Code"